tests/integration/test_db.py

```diff
@@ -377,6 +295,7 @@ async def test_extensions_blocking(ops_test: OpsTest, charm: str) -> None:
 
 @markers.juju2
 @pytest.mark.group(1)
+@pytest.mark.unstable
```

Comment:
Test fails consistently with:
```
AttributeError: 'NoneType' object has no attribute 'workload_status_message'
```
There doesn't seem to be a livepatch unit deployed. We should investigate further and reenable.

Reply:
Created [DPE-4989](https://warthogs.atlassian.net/browse/DPE-4989) to investigate and fix.

Reply:
Thanks!